maint/GenerateUcd.py

```diff
@@ -737,6 +737,13 @@ def write_bitsets(list, item_size):
     if x > 127 and x + other_case[x] < 128:
       other_case[x] = 0  
 
+# Append a couple of extra caseless sets (unreferenced by the record objects)
```

Comment:
the records existin the source file and were skipped (they have a T "type" and we only process the ones with "C" or "S".

I am confused about this though, as it would seem you are hardcoding the values everywhere else.

Reply:
I'll think about it. I'm still not sure we even want the Turkish characters in there, it's a temporary hack really.

Reply:
it is just a slightly cleaner hack IMHO; and even if somehow `OP_CHAR2` materializes, doing something more data driven and with less hardcoding would be easier to adjust.